ci/fedora-23.build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd /home &&
+mkdir build &&
+cd build &&
+cmake -DCMAKE_CXX_COMPILER=g++ .. &&
```

Comment:
Both .build files: to exit with error code if any warnings you can replace this line with `cmake -DCMAKE_CXX_FLAGS="-Werror" -DCMAKE_CXX_COMPILER=g++ .. &&` if I'm not mistaken.